src/Workspaces/Core/Portable/TemporaryStorage/TemporaryStorageServiceFactory.cs

```diff
@@ -20,474 +19,452 @@
 
 namespace Microsoft.CodeAnalysis.Host
 {
-    [ExportWorkspaceServiceFactory(typeof(ITemporaryStorageService), ServiceLayer.Default), Shared]
-    internal partial class TemporaryStorageServiceFactory : IWorkspaceServiceFactory
+    /// <summary>
+    /// Temporarily stores text and streams in memory mapped files.
+    /// </summary>
+    internal partial class TemporaryStorageService : ITemporaryStorageService2
```

Comment:
i wanted to rename this file.  but when i did, github lost that it was a rename and shows 500+ extra lines of add/remove.